argoverse/evaluation/stereo/model_analysis.py

```diff
@@ -0,0 +1,131 @@
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple, Union
+
+import numpy as np
+import torch
```

Comment:
I think we're going to need to put this in a try/catch statement since it's not a dependency of the package.